internal/frontend/token/token.go

```diff
@@ -91,33 +91,14 @@ func (T *Token) UintValue() (uint64, error) {
 	return strconv.ParseUint(string(T.Lit), 10, 64)
 }
 
+var (
```

Comment:
I'm not sure but are the variables holding compiled regular expressions need to be global ?

Reply:
of course it is not _required_ to be global

in the same time it _might_ be compiled once on package init and used everywhere later